timesketch/api/v1/resources/analysis.py

```diff
@@ -28,12 +30,23 @@
 from timesketch.lib.definitions import HTTP_STATUS_CODE_BAD_REQUEST
 from timesketch.lib.definitions import HTTP_STATUS_CODE_FORBIDDEN
 from timesketch.lib.definitions import HTTP_STATUS_CODE_NOT_FOUND
+from timesketch.lib.definitions import METRICS_NAMESPACE
 from timesketch.models.sketch import Analysis
 from timesketch.models.sketch import AnalysisSession
 from timesketch.models.sketch import Sketch
 from timesketch.models.sketch import Timeline
 
-
+# Metrics definitions
+METRICS = {
+    'analyzer_run': prometheus_client.Counter(
+        'timesketch_analyzer_run',
+        'Number of runs per analyzer',
+        ['name'],
+        namespace=METRICS_NAMESPACE
+    )
+}
```

Comment:
ok... qq... this looks like it just logs every use of an analyzer... do we want that to be pinned to a specific sketch, or do we want it to be `independent` of a sketch? or do we want both?

this is just a thought... not sure what you think....

Reply:
So in this first iteration this will just count every time an anlyzer is run, to get a holistic view on number of invocations. We can get more granular later.